azkustodata/query/column.go

```diff
@@ -0,0 +1,13 @@
+package query
+
+import "github.com/Azure/azure-kusto-go/azkustodata/types"
+
+// Column represents a column in a table.
+type Column interface {
```

Comment:
Why would such a simple class need an interface?

Reply:
To not have breaking changes if the internals change.